packages/core/data-transfer/lib/providers/local-file-source-provider.ts

```diff
@@ -78,8 +78,8 @@ class LocalFileSourceProvider implements ISourceProvider {
     // TODO: need to read the file & extract the metadata json file
     // => we might also need to read the schema.jsonl files & implements a custom stream-check
     const backupStream = this.#getBackupStream();
-
-    return this.#parseJSONFile<IMetadata>(backupStream, METADATA_FILE_PATH);
+    return null;
+    // return this.#parseJSONFile<IMetadata>(backupStream, METADATA_FILE_PATH);
```

Comment:
I think [this is almost ready](https://github.com/strapi/strapi/pull/14864), might be better to leave this in.

Reply:
yeah I commented it because it breaks the flow of the import, because the file doesn't exist